os/include/sys/syscall.h

```diff
@@ -443,27 +323,19 @@
 #define SYS_mq_timedsend               (__SYS_mqueue + 8)
 #define SYS_mq_unlink                  (__SYS_mqueue + 9)
 #define __SYS_environ                  (__SYS_mqueue + 10)
-#else
-#define __SYS_environ                  __SYS_mqueue
-#endif
 
 /* The following are defined only if environment variables are supported */
 
-#ifndef CONFIG_DISABLE_ENVIRON
 #define SYS_clearenv                   (__SYS_environ + 0)
 #define SYS_getenv                     (__SYS_environ + 1)
 #define SYS_putenv                     (__SYS_environ + 2)
 #define SYS_setenv                     (__SYS_environ + 3)
 #define SYS_unsetenv                   (__SYS_environ + 4)
 #define SYS_get_environ_ptr            (__SYS_environ + 5)
 #define __SYS_network                  (__SYS_environ + 6)
```

Comment:
We used `__SYS_XXX` to get proper offset regardless of enabled or disabled `CONFIG_XX`.
Even we remove the `CONFIG_XX`, do we still need it?
When we keep this, what is adv?